curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/meet-the-node-console.english.md

```diff
@@ -12,7 +12,11 @@ We recommend to keep the log panel open while working at these challenges. By re
 
 ## Instructions
 <section id='instructions'>
-Modify the <code>myApp.js</code> file to log "Hello World" to the console. 
+  
+* If you have not already done so, please read the instructions in [the introduction](/learn/apis-and-microservices/basic-node-and-express/) and start a new project on Glitch using [this link](https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-express/).
+  
+* Modify the <code>myApp.js</code> file to log "Hello World" to the console. 
+  
```

Comment:
Even though they are not causing a problem (per say), this line has two spaces instead of an empty line.  Might as well be consistent and remove the spaces.
```suggestion

```

Reply:
Thanks again, I'll check for this in the future.